tests/e2e-tests/src/orchestration.test.ts

```diff
@@ -20,7 +20,18 @@ describe('orchestration', () => {
         model_params: { max_tokens: 50, temperature: 0.1 }
       },
       prompt: {
-        template: [{ role: 'user', content: 'Hello!' }]
+        template: [
```

Comment:
Can this test be extended to check if the templating works as we expect it to?

Reply:
That's a good idea, done.